mars/services/task/execution/ray/config.py

```diff
@@ -77,6 +78,19 @@ def get_log_interval_seconds(self):
             "log_interval_seconds", DEFAULT_LOG_INTERVAL_SECONDS
         )
 
+    def get_check_slow_subtasks_interval_seconds(self) -> float:
+        return self._ray_execution_config.get(
```

Comment:
why no just reuse existing sppeculative config?

Reply:
This is not the speculative execution. If we implemented the speculative execution in Ray executor, then both of them can share the config.